src/ipld/cid_hashset.rs

```diff
@@ -2,27 +2,84 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use ahash::HashSet;
-use cid::Cid;
+use cid::{Cid, Version};
+use fvm_ipld_encoding::DAG_CBOR;
+use std::collections::BTreeSet;
 
+const BLAKE2B256: u64 = 0xb220;
```

Comment:
Can we import this constant from somewhere?

Reply:
It appears to be hidden.